sleap/gui/dialogs/formbuilder.py

```diff
@@ -224,25 +225,32 @@ def get_results(self) -> Optional[Dict[Text, Any]]:
         return self._results
 
 
-# We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1
 class ExponentialSpinBox(QtWidgets.QDoubleSpinBox):
+    """We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1"""
+
     def stepBy(self, steps: int):
+        """Overrides the default step behavior to increase or decrease by a factor of 10.
+
+        Args:
+            steps (int): The step count; positive values multiply by 10, negative values divide by 10.
+        """
         currValue = self.value()
 
-        if steps == 0:
-            return
-        
         if steps > 0:
-            newValue = currValue * 10
+            newValue = currValue * 10.0
             if currValue == 0:
                 newValue = 0.01
 
         elif steps < 0:
-            newValue = currValue * ((10) ** -1)
+            newValue = currValue * 0.1
 
-        # We need to keep it in the range (0 - 1000)
-        newValue = min(self.maximum(), newValue)  
+        else:
+            return
+
+        """Ensures value is in the range (0 - 1000)"""
```

Comment:
Triple quotes should only be used in multi-line comments (and ideally ONLY in docstrings).
```suggestion
        # Ensure value is in the range (0 - 1000)
```